aicsimageio/writers/ome_tiff_writer.py

```diff
@@ -507,7 +532,24 @@ def dim_or_1(dim: str) -> int:
         if channel_colors is not None:
             assert len(channel_colors) >= pixels.size_c
             for i in range(channel_count):
-                pixels.channels[i].color = Color(channel_colors[i])
+                this_channel_color_def = channel_colors[i]
+                if len(this_channel_color_def) != 3:
+                    raise ValueError(
+                        f"Expected RGB (3) color definition for channel color. "
+                        f"Received {len(this_channel_color_def)} values "
+                        f"({this_channel_color_def}) for image {image_index} "
+                        f"channel {i}."
+                    )
+                else:
+                    # Handle List[int] -> Tuple[int, int, int] for color def
+                    # Naive cast of tuple(List[int]) generates type: Tuple[int, ...]
+                    this_channel_color = (
+                        this_channel_color_def[0],
+                        this_channel_color_def[1],
+                        this_channel_color_def[2],
+                    )
```

Comment:
I think this can just be `tuple(this_channel_color_def)`

```python
[ins] In [1]: a = [1, 2, 3]

[ins] In [2]: tuple(a)
Out[2]: (1, 2, 3)
```

Reply:
See my comment:

```python
# Naive cast of tuple(List[int]) generates type: Tuple[int, ...]
```

I explicitely can't do that because of typing

Reply:
On the other hand, the way it is written ensures no more than 3 entries even if the color def had too many

Reply:
It will never reach that because I check for `len(this_channel_color_def) != 3` and raise

Reply:
yeah, forget i said anything 🤣

Reply:
Ah, fair. I guess the alternative could be `this_channel_color: Tuple[int, int, int] = tuple(this_channel_color_def)`?

Reply:
Nope, even that doesn't work. It raises a `expression has different type than expected value type` or something.